pkg/operator/ceph/object/spec.go

```diff
@@ -401,8 +418,14 @@ func (c *clusterConfig) makeDaemonContainer(rgwConfig *rgwConfig) (v1.Container,
 		container.VolumeMounts = append(container.VolumeMounts, mount)
 	}
 	if c.store.Spec.Gateway.CaBundleRef != "" {
-		updatedBundleMount := v1.VolumeMount{Name: caBundleUpdatedVolumeName, MountPath: caBundleExtractedDir, ReadOnly: true}
+		updatedBundleMount := v1.VolumeMount{Name: caBundleUpdatedVolumeName, MountPath: "/etc/ssl/combined-ca", ReadOnly: true}
 		container.VolumeMounts = append(container.VolumeMounts, updatedBundleMount)
+		container.Env = append(container.Env,
+			v1.EnvVar{
+				Name:  "SSL_CERT_FILE",
+				Value: "/etc/ssl/combined-ca/combined-ca-bundle.crt",
+			},
```

Comment:
This env var doesn't look like it's used anywhere. Is this intended?

Reply:
yes, it is like that. When SSL_CERT_FILE is set  Openssl/TLS/SSL libraries automatically get use of that variable.

Reply:
Got it. Could you make a short comment explaining that so it's clear that it's still needed even when unused directly?

Reply:
I added a comment .